Microsoft.Toolkit.Uwp.UI.Controls/MarkdownTextBlock/Parse/Inlines/ImageInline.cs

```diff
@@ -41,6 +41,12 @@ public ImageInline()
 
         public string Text { get; set; } = string.Empty;
 
+        /// <summary>
+        /// Gets or sets image width
+        /// If value is greater than 0, ImageStretch is set to UniformToFill
+        /// </summary>
+        public int ImageWidth { get; set; }
```

Comment:
should this be internal? does it make sense for this to have a non-private setter?